backend/src-common/src/main/java/org/eclipse/sw360/mail/MailUtil.java

```diff
@@ -191,10 +207,10 @@ private void sendMailWithSubjectAndText(String recipient, MimeMessage message) {
     private void writeMessageToLog(MimeMessage message) {
         try {
             log.info(String.format("E-Mail message dumped to log, because mailing is not configured [correctly]:\n"+
-            "From: %s\n"+
-            "To: %s\n"+
-            "Subject: %s\n"+
-            "Text: %s\n",
+                            "From: %s\n"+
```

Comment:
one good application of this would be to have the mails sent instead manually - so an admin could pull it out of the log. But in this case I am not sure to have the entire message printed.